libs/pageserver_api/src/models.rs

```diff
@@ -53,7 +53,7 @@ pub enum TimelineState {
     Suspended,
     /// A timeline is recognized by pageserver, but not yet ready to operate and not allowed to
     /// automatically become Active after certain events: only a management call can change this status.
```

Comment:
Whole rustdoc seems obsolete now.

Reply:
Indeed, the comments seem very unclear now. Opened https://github.com/neondatabase/neon/issues/2961 for that.